src/SingleProject/Resizetizer/test/UnitTests/BaseTest.cs

```diff
@@ -14,7 +14,7 @@ public class BaseTest : IDisposable
 	{
 		private const string TestFolderName = "Microsoft.Maui.Resizetizer.Tests";
 		private const string TestImagesFolderName = "imageresults";
-		private const double ImageErrorThreshold = 0.0027;
+		private const double ImageErrorThreshold = 0.27;
```

Comment:
What do we think about this diff @mattleibow  @BretJohnson  ?

Reply:
@anandhan-rajagopal Could you share more details? Why required the change?